docs/evm-sidechain/xrpl-evm-sidechain-faq.md

```diff
@@ -0,0 +1,104 @@
+
+
+### How many EVM sidechains will there be? Can I set up my own?
+
+There will be one public EVM sidechain supported by Ripple and PeerSyst for the XRPL community. This EVM sidechain will have multiple validators run by XRPL community members. The software needed to run an EVM sidechain will be open source and any community member can use it to set up their own EVM sidechains, for permissioned use cases for example.
+
+
+### What are any critical dependencies to the launch of the EVM sidechain?
+
+The main dependency is that the amendment for cross-chain bridges (XLS-38d) proposal needs to be approved by the community and be live on the XRPL Mainnet. This is because the bridge between the XRPL and the EVM sidechain uses the XLS-38d bridge design.
+
+
+### Who will run the XRPL - EVM sidechain bridge?
+
+The bridge connecting the XRPL and the EVM sidechain will use the XLS-38d standard and will be operated by XRPL community members by running witness servers.
+
+
+### What does success look like for the EVM sidechain after launch on XRPL Mainnet?
+
+Long term, the EVM sidechain has a large and sustainable ecosystem of developers. The XRPL developer ecosystem has adopted the EVM sidechain as an extension of the XRPL and uses it as a testing ground before features are proposed on XRPL Mainnet or has deployed apps there that are not suitable for the XRPL Mainnet.
+
+
+### Who is our target user for the EVM sidechain?
+
+An EVM application developer who has experience writing Solidity applications and who is looking to monetize their applications on other EVM compatible blockchains outside of Ethereum. This developer isn’t happy about the high gas fees and low transaction speeds on Ethereum during peak usage.
+
+Another segment of EVM developers is the EVM core developer that works on new protocol primitives on Ethereum or similar blockchains. They have less opportunities to contribute to Ethereum so the EVM sidechain provides them a blank slate to add new innovative capabilities.
+
+
+### What consensus algorithm does the EVM sidechain use?
+
+The EVM sidechain will use the Proof of Authority (PoA) consensus algorithm. A set of validators will confirm blocks on the EVM sidechain. New validators can join, existing validators can leave or based on voting by the existing set of validators.
+
+
+### Why should an EVM developer use the EVM sidechain of the XRPL as opposed to Ethereum or any other EVM compatible blockchain?
+
+An EVM developer that builds on the EVM sidechain essentially gets a blank slate to start from. There will be considerable excitement to deploy the first apps that were successful on other EVM blockchains on the EVM sidechain. The EVM sidechain has a 3.5 second block time  which is much lower than the 12-15 seconds used on Ethereum. XRP is one of the most liquid cryptocurrencies, and developers can reach millions of XRP holders by launching dapps on the EVM sidechain where XRP is a native gas token.
```

Comment:
An EVM developer who builds on the EVM sidechain essentially starts from a blank slate. Developers will deploy apps that have been successful on other EVM blockchains on the EVM sidechain. The EVM sidechain has a 3.5 second block time, which is much lower than the 12-15 seconds taken on Ethereum. XRP is one of the most liquid cryptocurrencies. Developers can reach millions of XRP holders by launching dapps on the EVM sidechain, where XRP is a native gas token.